domains/client/domain-executor/README.md

```diff
@@ -19,3 +36,54 @@ Each executor instance, whether for system domain or core domain, consists of th
   - Executor components specfic to the system domain.
 - `core_{block_processor,bundle_producer,bundle_proposer,worker}.rs`/`core_gossip_message_validator`
   - Executor components specfic to the core domain.
+
+## Run a local testnet
+
+Clone the repo and build the executables:
+
+```bash
+git clone https://github.com/subspace/subspace
+cd subspace
+cargo build --release --bin subspace-farmer --bin subspace-node
+```
+
+Start a local testnet:
+
+```bash
+# Run a primary chain, a system domain and a core domain node in one command:
+subspace-node --dev -d tmp --ws-port 4567 -- --alice --dev --ws-port 5678 -- --domain-id 1 --alice --dev --ws-port 6789
```

Comment:
I'd rather recommend `cargo run` here. No need to build anything upfront. This is especially confusing given above instructions because `subspace-node` is no in `PATH`. `cargo run` will always give the correct result. For local testing if you know what you're doing you can build and run manually if you want.

`-d tmp` is also not necessary, same for `--ws-port` and `--node-rpc-url` below.